Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5479,16 +5531,41 @@ LayoutUnit RenderBox::computeLogicalWidthFromAspectRatio(RenderFragmentContainer
 
 std::pair<LayoutUnit, LayoutUnit> RenderBox::computeMinMaxLogicalWidthFromAspectRatio() const
 {
-    LayoutUnit blockMinSize = constrainLogicalHeightByMinMax(LayoutUnit(), std::nullopt);
-    LayoutUnit blockMaxSize = constrainLogicalHeightByMinMax(LayoutUnit::max(), std::nullopt);
+    const RenderStyle& styleToUse = style();
```

Comment:
I think styleToUse is not needed, the getter should get optimized. Also this nicely will make the ASSERT the first statement.